docs/search/extensibility/create_custom_aggregation.md

```diff
@@ -106,15 +106,17 @@ and provide it with the aggregation class in the `aggregationClass` parameter.
 
 === "Solr"
 
-    Tag the service with `ibexa.solr.query.location.aggregation_result_extractor`.
+    Tag the service with `ibexa.search.solr.query.location.aggregation.result.extractor`.
 
     ``` yaml
-    App\Query\Aggregation\Solr\PriorityAggregationResultExtractor:
+    app.search.solr.query.aggregation_result_extractor.priority_range_aggregation:
         class: Ibexa\Solr\ResultExtractor\AggregationResultExtractor\RangeAggregationResultExtractor
         arguments:
             $aggregationClass: 'App\Query\Aggregation\PriorityRangeAggregation'
```

Comment:
I tested again, but I am still getting some errors, sorry I missed this yesterday, with these two changes everything works :)
```suggestion
            $aggregationClass: 'App\Query\Aggregation\Solr\PriorityRangeAggregation'
```